dhcp_lease.yml

```diff
@@ -3,11 +3,25 @@
   gather_facts: false
 
   vars:
-    x_port_id: "{{ port_id | regex_replace('[^a-z0-9_]', '') }}"
+    x_nic_id: "{{ nic_id | regex_replace('[^a-z0-9-]', '') }}"
     x_mac_address: "{{ mac_address | regex_replace('[^0-9a-fA-F:]', '') }}"
     x_ip_address: "{{ ip_address | regex_replace('[^0-9.]', '') }}"
 
   tasks:
+    - name: Find port_id by nic_id
+      ansible.builtin.shell: >
+        ovn-nbctl find logical_switch_port external_ids:ovirt_device_id="{{ x_nic_id }}" | grep "^name" | cut -d "\"" -f 2
```

Comment:
maybe use port _uuid, it's always first in csv output. then can be used in ovn-nbctl command instread of logical_switch_port name.
```
[root@engine-ovirt ~]# ovn-nbctl --bare -f csv find logical_switch_port external_ids:ovirt_device_id="d75d0e14-f519-497a-892a-4cf64f152ad0" | cut -d, -f1
32bcbf9e-3e44-41e1-87a5-8b64e5264907
```

add waiting from ansible side, because VM should be started before logical-switch-port appears